src/init.cpp

```diff
@@ -744,6 +744,7 @@ void InitParameterInteraction(ArgsManager& args)
         args.SoftSetArg("-bytespersigopstrict", "0");
         args.SoftSetArg("-permitbarepubkey", "1");
         args.SoftSetArg("-permitbaremultisig", "1");
+        args.SoftSetArg("-rejectparasites", "0");
```

Comment:
The change to the DEFAULT_REJECT_PARASITES should be part of this commit too

Reply:
but I can't put the same change on two commit?

Reply:
Add it as `false`, then change to `true` in this commit